src/css/bootstrap-overrides.scss

```diff
@@ -7,6 +7,27 @@
   position: absolute;
 }
 
+header {
+  padding-left: 15px;
+  padding-right: 15px;
+  border-bottom: 1px solid #EEF2F5;
```

Comment:
Instead of adding the border here, you can use the bootstrap class `border-bottom`. It will add the border with the same style as everywhere so you don't have to get the color yourself. (https://getbootstrap.com/docs/4.0/utilities/borders/#border)